src/test/java/org/cactoos/io/Sha256DigestOfTest.java

```diff
@@ -74,10 +74,12 @@ public void checksumFromFile() throws Exception {
             "Can't calculate the file's SHA-256 checksum",
             new HexOf(
                 new Sha256DigestOf(
-                    new InputOf(
-                        new ResourceOf(
-                            "org/cactoos/digest-calculation.txt"
-                        ).stream()
+                    new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.